server/server.go

```diff
@@ -41,7 +41,6 @@ func setupRouter(ctx context.Context, logger *zerolog.Logger) (context.Context,
 
 	r.Use(chiware.Timeout(60 * time.Second))
 	r.Use(middleware.BearerToken)
-	r.Use(middleware.RateLimiter(ctx))
```

Comment:
I would suggest employing some form of rate limiting (figure out what is applicable for this use case)  This will help in the long run when someone is attempting to DOS (inadvertently or on purpose).  Can track in another issue.

Reply:
Open https://github.com/brave/go-sync/issues/3 to track this.